subt/docker/robotika/ros_proxy_node.cc

```diff
@@ -524,6 +583,7 @@ int main(int argc, char** argv)
       }
       ros::spinOnce();
       loop_rate.sleep();
+      controller.sendLogPart();
```

Comment:
It could happen, that during `spinOnce` more than 1000 bytes of data appeared in the logfile. In that case the /robot_data topic would permanently fall behind and the end of the log file would not necessarily be in /robot_data.

Reply:
Yes, I am aware of that. On the other hand I am not sure what strategy would be reasonable (at the moment we do not even know when we can start logging). In former version I was dumping logfile AFTER the exploration was completed to make sure that I do not lose any resources during explore ... so I guess that this will depend on AWS capabilities. We have 2GB/1hour of simulation so it is approximately 1MB/sec what we can store. The loop at the moment runs at 20Hz -> 50kB is max we could save "per frame". I would increase the buffer size to 1MB as soon as we know when we can start safely log (now too fast logging would loose too much data).